scripts/deploy/deploy.py

```diff
@@ -69,7 +69,7 @@ def get_head_sha():
   with open(os.devnull, 'w') as devnull:
     try:
       build_cmd = 'rm -rf dist;' \
-                  './pants binary scripts/sdk:intellij-pants-plugin-publish'
+                  '.cache/pants-host/pants binary scripts/sdk:intellij-pants-plugin-publish'
```

Comment:
Why?

Reply:
That's because I had to use pants with pumped Zinc